### PR TITLE
[Guide] Discover data guide

### LINKED ISSUE #996: [Doc - Guides] - Discover Data
[EPIC] https://github.com/CartoDB/cartoframes/issues/967

### PR DESCRIPTION
**Pull request by alrocar**
Closes #996

### CODE REVIEW

**Comment by alrocar** [comment]
http://cd-reference-enrichment-catalog.developers.carto-staging.com/developers/cartoframes/guides/Explore-the-Data-Observatory-catalog/

Pending stuff:

- Would be nice having a scroll in the dataframe and Dataset, Variable prints. I don't think we should cut them (doing a head or ellipsis), since the whole purpose of this guide is understanding the available data in the Catalog.
- Still have to extract the map as an external frame and include it with an iframe.

File: docs/developer-center/guides/06-Explore-the-Data-Observatory-catalog.md [hunk 1]
```diff
@@ -0,0 +1,5614 @@
...
+      <td>None</td>
+    </tr>
+    <tr>
+      <th>76</th>
+      <td>SUM</td>
+      <td>POPCYGRP</td>
+      <td>carto-do.ags.demographics_sociodemographic_usa...</td>
+      <td>INTEGER</td>
+      <td>Population in Group Quarters (2019A)</td>
+      <td>carto-do.
```
Comment (elenatorro) [inline docs/developer-center/guides/06-Explore-the-Data-Observatory-catalog.md]: We need to fix this in our markdown. The subscription contract is an auto generated HTML. What I would to is to **take an screenshot** from the notebook and use it here, what do you think?

Comment (elenatorro) [inline docs/developer-center/guides/06-Explore-the-Data-Observatory-catalog.md]: The image path is not correct and it's not rendering

Comment (elenatorro) [inline docs/developer-center/guides/06-Explore-the-Data-Observatory-catalog.md]: I'm adding a new utility class named `u-vertical-scroll`. Please add it in the table, next to the "dataframe" class (I'll add this information in the guidelines asap)

File: docs/developer-center/guides/06-Explore-the-Data-Observatory-catalog.md [hunk 2]
```diff
@@ -0,0 +1,5614 @@
...
+from cartoframes.data.observatory import Dataset
+dataset = Dataset.get('ags_sociodemogr_e92b1637')
+variables = dataset.variables
+variables
+```
+
+
+
+
+    [<Variable.get('HINCYMED65_310bc888')> #'Median Household Income: Age 65-74 (2019A)',
```
Comment (elenatorro) [inline docs/developer-center/guides/06-Explore-the-Data-Observatory-catalog.md]: This piece of code is also quite long. Do you think it'd be ok to  ✂️cut  ✂️it a bit, and have something like:

```
[<Variable.get('HINCYMED55_1a269b4b')> #'Median Household Income: Age 55-64 (2019A)',
<Variable.get('HINCYMED55_1a269b4b')> #'Median Household Income: Age 55-64 (2019A)',
<Variable.get('HINCYMED55_1a269b4b')> #'Median Household Income: Age 55-64 (2019A)',
...
<Variable.get('HINCYMED55_1a269b4b')> #'Median Household Income: Age 55-64 (2019A)',
<Variable.get('HINCYMED55_1a269b4b')> #'Median Household Income: Age 55-64 (2019A)',
<Variable.get('HINCYMED55_1a269b4b')> #'Median Household Income: Age 55-64 (2019A)',]
```

Another option would be to put this inside a `<div>` with the `u-vertical-scroll` class (the same we'll be using in the tables), but that might require more adjustments.

File: docs/developer-center/guides/06-Explore-the-Data-Observatory-catalog.md (hunk 1 above)
Comment (alrocar) [reply docs/developer-center/guides/06-Explore-the-Data-Observatory-catalog.md]: You are right, forgot about that 👍

File: docs/developer-center/guides/06-Explore-the-Data-Observatory-catalog.md (hunk 2 above)
Comment (alrocar) [reply docs/developer-center/guides/06-Explore-the-Data-Observatory-catalog.md]: yep, I prefer the scroll thing, I'll add it. Thanks!

**Comment by alrocar** [comment]
summoning @makella in case you are available for doing some proof-reading, grammar or redaction fixes/suggestions this week 🙏

**Comment by elenatorro** [comment]
As discussed in the meeting, let's merge them to develop and check the guides all together in a final review ✨ awesome work!!!

**Pull request merged**